azurerm/config.go

```diff
@@ -595,6 +596,13 @@ func (c *ArmClient) registerDatabases(endpoint, subscriptionId string, auth auto
 	sqlADClient.Sender = sender
 	sqlADClient.SkipResourceProviderRegistration = c.skipProviderRegistration
 	c.sqlServerAzureADAdministratorsClient = sqlADClient
+
+	sqlVNRClient := sql.NewVirtualNetworkRulesClientWithBaseURI(endpoint, subscriptionId)
+	setUserAgent(&sqlVNRClient.Client)
+	sqlVNRClient.Authorizer = auth
+	sqlVNRClient.Sender = sender
+	sqlVNRClient.SkipResourceProviderRegistration = c.skipProviderRegistration
+	c.sqlVirtualNetworkRulesClient = sqlVNRClient
```

Comment:
_minor_ can we switch this over to using the new registration syntax? e.g. `c.configureClient(&sqlVNRClient.Client, auth)

Reply:
👍